apps/central-scan/frontend/src/app.test.tsx

```diff
@@ -27,20 +25,27 @@ import {
 } from '../test/react_testing_library';
 import { App } from './app';
 import { MachineConfigResponse } from './config/types';
-import { createMockApiClient, MockApiClient, setAuthStatus } from '../test/api';
+import {
+  createMockApiClient,
+  MockApiClient,
+  setAuthStatus,
+  setUsbDriveStatus,
+} from '../test/api';
+import { mockUsbDriveStatus } from '../test/helpers/mock_usb_drive';
 
 let mockApiClient: MockApiClient;
 
 beforeEach(() => {
   jest.restoreAllMocks();
 
-  window.kiosk = undefined;
-
```

Comment:
AFAICT, `central-scan`'s only use of `window.kiosk` are now the reboot methods and setting the date and time. These could easily be moved to `libs/backend`, and `central-scan` could be an initial test case for a `kiosk-browser` alternative.

Reply:
It's the only app that does not print.

Reply:
That's awesome!